src/views/v2/month/calendar-body/day/multiday-events/multiday-event.php

```diff
@@ -59,32 +63,56 @@
 	if ( $event->dates->end->format( 'Y-m-d' ) < $today_date ) {
 		$classes[] = 'tribe-events-calendar-month__multiday-event--past';
 	}
-
-
-} else {
-	$classes[] = 'tribe-events-calendar-month__multiday-event--hidden';
 }
 
 ?>
 <div class="tribe-events-calendar-month__multiday-event-wrapper">
 	<article <?php tribe_classes( $classes ); ?> data-event-id="<?php echo esc_attr( $event->ID ); ?>">
-		<time
-			datetime="<?php echo esc_attr( $event->dates->start->format( Dates::DBDATEFORMAT ) ); ?>"
-			class="tribe-common-a11y-visual-hide"
-		>
-			<?php echo esc_attr( $event->dates->start->format( Dates::DBDATEFORMAT ) ); ?>
-		</time>
-		<a href="<?php echo esc_url( $event->permalink ) ?>" class="tribe-events-calendar-month__multiday-event-inner">
-			<?php if ( $event->featured ) : ?>
-				<em
-					class="tribe-events-calendar-month__multiday-event-featured-icon tribe-common-svgicon tribe-common-svgicon--featured"
-					aria-label="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
-					title="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
-				></em>
+		<div class="tribe-events-calendar-month__multiday-event-hidden">
+			<time
+				datetime="<?php echo esc_attr( $event->dates->start->format( Dates::DBDATEFORMAT ) ); ?>"
+				class="tribe-common-a11y-visual-hide"
+			>
+				<?php echo esc_attr( $event->dates->start->format( Dates::DBDATEFORMAT ) ); ?>
+			</time>
+			<a
+				href="<?php echo esc_url( $event->permalink ); ?>"
+				<?php if ( $display_tooltip ) : ?>
+					data-js="tribe-events-tooltip"
+					data-tooltip-content="#tribe-events-tooltip-content-<?php echo esc_attr( $event->ID ); ?>"
+					aria-describedby="tribe-events-tooltip-content-<?php echo esc_attr( $event->ID ); ?>"
+				<?php endif; ?>
+			>
+				<?php if ( $event->featured ) : ?>
+					<em
+						class="tribe-events-calendar-month__multiday-event-featured-icon tribe-common-svgicon tribe-common-svgicon--featured"
+						aria-label="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
+						title="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
+					></em>
+				<?php endif; ?>
+				<h3 class="tribe-events-calendar-month__multiday-event-title tribe-common-h8">
+					<?php echo wp_kses_post( get_the_title( $event->ID ) ); ?>
+				</h3>
+			</a>
+		</div>
+		<?php if ( $should_display ) : ?>
+			<div class="tribe-events-calendar-month__multiday-event-bar">
+				<div class="tribe-events-calendar-month__multiday-event-inner">
```

Comment:
i'd like to see this class name match the structure it's in, so since we're under the `__multiday-event-bar` class, we should have this as `__multiday-event-bar-inner` or something similar.